apps/app-article-server/src/browser/components/article.js

```diff
@@ -31,6 +32,7 @@ class Article extends Component {
       document.getElementsByTagName("HTML")[0].setAttribute("data-theme", "dark");
     }
     this.getMostReadArticles();
+    this.getAdvertorialLiftup();
```

Comment:
Is this the correct way of implementing lazy loading?

Reply:
Looks like you need to import it lazily, the same way as with MostReadArticles on line 13.

Reply:
Actually I would say it would be best to use `react-lazy-load-image-component` library the same way it is used in article-content for the image the advertorial-liftup-component. So:

`import { LazyLoadImage } from "react-lazy-load-image-component";`

and 

```
<LazyLoadImage 
  className={"adv-image"}
  src={props.image}
  threshold="200"
  effect="opacity"
  alt=""
/>
 ```
 
 I'm not sure React.lazy does much when it's returned from the server.